client/src/commons/ObserverDP/Observer.ts

```diff
@@ -0,0 +1,27 @@
+class Observer {
+
+    _observants: Map<number, () => void>;
+
+    constructor () {
+        this._observants = new Map<number, () => void>();
+    }
+
+    subscribe(id: number, gotObserverUpdate: () => void) {
+        this._observants.set(id, gotObserverUpdate);
+    }
+
+    notifyAll() {
+        this._observants.forEach(notification => notification());
+    }
+
+    notifySpecific(id: number) {
```

Comment:
I think calling the function `notify` is clear enough, considering there is also a `notifyAll` function.